src/teleport.cpp

```diff
@@ -21,6 +21,7 @@
 
 #include "teleport.h"
 #include "game.h"
+#include <boost/format.hpp>
```

Comment:
Not sure if this doesn't require explicitly adding this library to cmake.

Reply:
boost::format is not required at all!